src/modules/dataCenter.js

```diff
@@ -0,0 +1,10 @@
+import axios from 'axios'
+
+axios.defaults.baseURL = 'http://localhost:3000'
```

Comment:
Please put this in your `index.js` file and add the namespace `/api/` to it as well since you at the moment have that namespace for everything in your API

Reply:
Okay, Thomas told us oppositely, but we don't mind either!